### PR TITLE
Add hie exclusions

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
Add HIE exclusion measures and description of `additional-measures.json` in README.md

[QPPA-419](https://jira.cms.gov/browse/QPPA-419)

### CODE REVIEW

**Comment by ivanang** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -81,3 +81,11 @@ When making changes to measures-data, include tests in the tests directory and m
 ```
 npm run test
 ```
+
+### Additional measures
+
+`util/additional-measures.json` includes data objects which are necessary for scoring but are not MIPS measures. At this time `util/additional-meausures.json` includes:
```
Comment (kencheeto) [inline README.md]: `meausures` -> `measures`

File: util/additional-measures.json [hunk 2]
```diff
@@ -80,5 +80,33 @@
...
     "objective": "electronicPrescribing"
-  }
+  },
+  {
+    "category": "aci",
+    "firstPerformanceYear": 2017,
+    "lastPerformanceYear": null,
+    "metricType": "boolean",
+    "measureId": "ACI_LVSSC_1",
+    "title": "Low-Volume Send a Summary of Care",
```
Comment (marimiyachi) [inline util/additional-measures.json]: Is there an email where these titles are coming from? I'm finding the titles confusing. What about "Low-Volume Outgoing Transfer of Care" and "Low-Volume Incoming Transfer of Care"

File: README.md (hunk 1 above)
Comment (marimiyachi) [inline README.md]: awesome!

Comment (abarciauskas-bgse) [inline README.md]: Thanks for catching that ! https://github.com/CMSgov/qpp-measures-data/pull/18/commits/695b639e6639eb87ddc066e9cfc723b8b715215d

File: util/additional-measures.json (hunk 2 above)
Comment (abarciauskas-bgse) [inline util/additional-measures.json]: These came from the text in the ticket, where "Send a Summary of Care" and "Request / Accept a Summary of Care" appear to be "titles" of the original measures: [QPPA-419](https://jira.cms.gov/browse/QPPA-419)

But I like your titles better still so updated: https://github.com/CMSgov/qpp-measures-data/pull/18/commits/9a4908daed544e55e8c05c715fd0c5b1ecca2682

**Pull request closed without merging**